monero-outreach-docs/translations/es/archive/triptych_es.md

```diff
@@ -0,0 +1,43 @@
+
+Triptych supera varios desafíos para la aplicación de las firmas de anillos en Monero. Provee de vinculación. La técnica de las firmas de anillo en Monero tieneque bloquear la reutilización de llaves públicas legítimas para impedir gastar un TXO más de una vez (lo que conocemos como doble gasto). Triptych hace esto utilizando etiquetas vinculables que son similares a las llaves de imagen que se utilizan hoy y también integra las transacciones confidenciales que ocultan las cantidades en Monero. Datos especiales llamados llaves de compromiso permiten comprobar que las cantidades de Monero en una transacción son reales sin revelarlas. Las llaves de compromiso con las llaves de firma y las etiquetas vinculadas, son los tres componentes que originan el nombre Triptych, que significa algo de tres partes.
+
+### _Probando alternativas_
+
+En la práctica Triptych se ejecuta rápido. Sarang Noether, uno de los creadores, reportó que al momento de las pruebas, la ejecución del código mostró notables mejoras. Al usar más o menos 63 decoys con Triptych, el proceso se ejecuta 10% más rápido que el usar 10 decoys con CLSAG. Estos resultados muestran claras mejoras a Monero que también serán prácticas al momento de la integración.
+
+Sin embargo, aún queda trabajo para que Triptych ingrese al código base de Monero ya que se requiere de una validación exhaustiva. Aunque el documento del algoritmo de Triptych ha sido revisado y aceptado por el taller internacional ESORICS CBT 2020, Triptych aún puede beneficiarse de las modificaciones e implementaciones en el software que se le hagan a estos algoritmos. La implementación incluso puede ser optimizada aún más para reducir los tiempos de validación. Dedicarle este esfuerzo a Triptych reemplaza el trabajo que se le puede dedicar a otras alternativas.
+
+Una de esas alternativas, vista con buenos ojos, es la versión avanzada de Triptych llamada Arcturus, desarrollada por Sarang Noether. Arcturus reduce aún más el tamaño de las transacciones utilizando una única firma criptográfica de datos agrupada para toda la transacción en vez de una independiente para cada TXO de entrada. Arcturus y Triptych tienen tiempos de validación similares. Lo que se gana en el tamaño de la transacción en Arcturus es un algoritmo que requiere de una suposición matemática que no es necesaria para Triptych.
+
+### _En resumen_
+
+Con Triptych y su extensión Arcturus, Monero tiene una tecnología a la mano con un gran potencial para reducir el crecimiento del tamaño de los datos de las firmas de anillo, de linealmente a escala logarítmica (incluyendo los decoys utilizados para ocultar al emisor). Si es acogido, le permitiría al número de decoys incrementarse sin incrementar el tamaño de la cadena de bloques o sin incrementar las validaciones que tenga que hacer la máquina. La tecnología de Triptych, mantiene el progreso ininterrummpido de Monero, exhibiendo lo posible y protegiendo la privacidad y la libertad de sus usuarios.
```

Comment:
And the last one:

_La tecnología de Triptych, mantiene el progreso **ininterrumpido** de Monero, exhibiendo lo posible y protegiendo la privacidad y la libertad de sus usuarios._

Reply:
Fixed :)